README.md

```diff
@@ -11,7 +11,11 @@ This script creates a new Custom role "Custom Role - Cloud One Conformity" withi
 This new Custom role along with the built-in "Reader" role will be applied to either all the Subscriptions in the
 Active Directory or just the subscription that is specified.
 
-The script can also be used to create a new App Registration with the default name "Conformity Azure access".
+The script can also create a new App Registration to apply these permissions which can be used to allow Conformity to access to your Azure instance.
+
+### Minimum Azure CLI version
+
+The version of Azure CLI used to run the script must be `2.37.0` or higher
```

Comment:
🔧 We should increase this to `2.40.0` as that is the version we developed against.

```suggestion
The version of Azure CLI used to run the script must be `2.40.0` or higher
```